docs/UserGuide.adoc

```diff
@@ -219,22 +230,38 @@ Medicines with quanities of less than or equal to 20 will be displayed in the wa
 
 === Exporting as CSV file format : `export`
 
-You can export the current Medicine Inventory data shown in the GUI as CSV file format. +
+You can export the current Medicine Inventory data shown in the GUI as Comma-separated values (CSV) file format which is supported by many commonly used spreadsheet applications such as Microsoft Excel (Windows), LibreOffice (Linux) or Numbers (MacOS).
+
+This feature is especially useful if you want to save the current Medicine Inventory data in a file so that it can be printed later or if you want to organise the Medicine Inventory data using a spreadsheet application for you, your superior or your supplier's reference.
+
+In addition, you can even use the spreadsheet application to convert the CSV file to Microsoft Excel format and use all the functionalities provided by Microsoft Excel such as drawing graphs or chart to display the overview of your clinic's medicine inventory. +
 Format: `export [FILE_NAME]`
 
 ****
-* If no file name is specified, the export will just name the file according to the date and time of export.
+* The `FILE_NAME` field is optional. You can refer to <<Appendix A: File Naming Convention>> for supported file name format if you want to specify a file name for the exported CSV file.
```

Comment:
FILE_NAME is optional is already expressed by the square brackets

Reply:
True that is a valid point though I think I read above some example features their description state at least one or more optional parameter is required so I thought despite indicating earlier that square brackets is optional I still mention it as a note in the description